recipes/python-chromedriver-binary/bld.bat

```
@@ -0,0 +1,5 @@
+python download-chromedriver
+7z x chromedriver.zip -ochromedriver
```

Also please add a `if errorlevel 1 exit 1` line here so it fails early

---

This new conda-forge linter is pretty good

---

😆